demo/screens/SellOrderClaimScreen.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import { claimSellOrder } from "../actions";
+import SellOrderClaim from "@presentational/SellOrderClaim";
+
+export class SellOrderClaimScreen extends React.Component {
+  _onClaim = sellOrder => {
+    const { claimSellOrder } = this.props;
+
+    claimSellOrder(sellOrder);
```

Comment:
Is this function sync or async?

Reply:
sync this dispatch functions is used to store which land users want to buy.

Reply:
Okay, thanks. Resolving this.